src/System.Windows.Forms/src/System/Windows/Forms/TextBoxBase.TextBoxBaseAccessibleObject.cs

```diff
@@ -21,6 +21,31 @@ public TextBoxBaseAccessibleObject(TextBoxBase owner) : base(owner)
                 UseTextProviders(_textProvider, _textProvider);
             }
 
+            internal void ClearObjects()
+            {
+                _textProvider = null;
+                _owningTextBoxBase = null;
+                ClearTextProviders();
+
+                // A place for future memory leak fixing:
+                //
+                // 1) This method should be added to the ControlAccessibleObject class:
+                //    internal void ClearOwnerControl()
+                //    {
+                //        this.Owner = null;
+                //    }
+                //
+                // 2) Owner property shoud be changed from this
+                //        public Control Owner { get; }
+                //    to something like this
+                //        public Control? Owner { get; private set; }
+                //
+                // 3) These changes will produce different sorts of warnings:
+                //     non-nullable member will became nullable, additional checks for null should be added, etc.
+                // 4) This method call should be uncommented
+                //        ClearOwnerControl();
```

Comment:
Not sure if I missed it earlier

```suggestion
                // 3) These changes will produce different sorts of warnings:
                //     non-nullable member will became nullable, additional checks for null should be added, etc.
                //
                // 4) This method call should be uncommented
                //        ClearOwnerControl();
```